packages/eslint-plugin/src/configs/recommended.json

```diff
@@ -2,35 +2,41 @@
   "extends": "./configs/base.json",
   "rules": {
     "@typescript-eslint/adjacent-overload-signatures": "error",
-    "@typescript-eslint/array-type": "error",
+    "@typescript-eslint/await-thenable": "error",
+    "@typescript-eslint/ban-ts-ignore": "error",
     "@typescript-eslint/ban-types": "error",
     "camelcase": "off",
     "@typescript-eslint/camelcase": "error",
     "@typescript-eslint/class-name-casing": "error",
-    "@typescript-eslint/consistent-type-definitions": "error",
     "@typescript-eslint/explicit-function-return-type": "warn",
-    "@typescript-eslint/explicit-member-accessibility": "error",
-    "indent": "off",
-    "@typescript-eslint/indent": "error",
     "@typescript-eslint/interface-name-prefix": "error",
     "@typescript-eslint/member-delimiter-style": "error",
     "@typescript-eslint/no-angle-bracket-type-assertion": "error",
     "no-array-constructor": "off",
     "@typescript-eslint/no-array-constructor": "error",
+    "@typescript-eslint/no-empty-function": "error",
     "@typescript-eslint/no-empty-interface": "error",
     "@typescript-eslint/no-explicit-any": "warn",
+    "@typescript-eslint/no-for-in-array": "error",
     "@typescript-eslint/no-inferrable-types": "error",
     "@typescript-eslint/no-misused-new": "error",
+    "@typescript-eslint/no-misused-promises": "error",
     "@typescript-eslint/no-namespace": "error",
-    "@typescript-eslint/no-non-null-assertion": "error",
+    "@typescript-eslint/no-non-null-assertion": "warn",
     "@typescript-eslint/no-object-literal-type-assertion": "error",
-    "@typescript-eslint/no-parameter-properties": "error",
+    "@typescript-eslint/no-this-alias": "error",
+    "@typescript-eslint/no-unnecessary-type-assertion": "error",
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": "warn",
     "no-use-before-define": "off",
     "@typescript-eslint/no-use-before-define": "error",
     "@typescript-eslint/no-var-requires": "error",
+    "@typescript-eslint/prefer-includes": "error",
     "@typescript-eslint/prefer-namespace-keyword": "error",
-    "@typescript-eslint/type-annotation-spacing": "error"
+    "@typescript-eslint/prefer-regexp-exec": "error",
+    "@typescript-eslint/prefer-string-starts-ends-with": "error",
+    "@typescript-eslint/require-await": "error",
```

Comment:
Should we disable [require-await](https://eslint.org/docs/rules/require-await) rule?

Reply:
definitely not - with the changes that were introduced to it (it allows you to have an async function that _returns_ a promise), I don't see why you wouldn't want to use this rule.

[Considering how much extra code can be generated when you create an `async` function with no awaits](https://www.typescriptlang.org/play/?target=3#code/IYZwngdgxgBAZgV2gFwJYHsL3egFAShgG8BfIA), best practices dictate you shouldn't do it.

Reply:
My intention was that, should we disable `require-await` because we enabled `@typescript-eslint/require-await` if the two are duplicated?